client/src/js/components/ViewsAlert.jsx

```diff
@@ -3,12 +3,12 @@ import React from 'react';
 const ViewsAlert = (props) => {
   if (props.data > 200) {
     return (
-      <hgroup id="viewsAlert" className="lightBorder" >
-        <div className="viewsAlertText" >
-          <div className="title_1" >{'This home is on people\'s minds.'}</div>
-          <div id="ownerName" >{`It's been viewed ${props.data}+ times in the past week.`}</div>
+      <hgroup id="viewsAlert" className="lightBorder">
+        <div className="viewsAlertText">
+          <div className="title_4" >{'This home is on people\'s minds.'}</div>
```

Comment:
Why do you need the text in brackets?

Reply:
got the eslint, html entities must be scaped,  error due to the single quote in the text.